cylc/flow/scripts/cat_log.py

```diff
@@ -367,28 +436,53 @@ def main(
     except KeyError:
         mode = options.mode
 
+    if tokens and tokens.get('cycle') and not tokens.get('task'):
+        print('Please provide a workflow, task or job ID', file=sys.stderr)
+        sys.exit(1)
```

Comment:
Fixes a small bug where requesting logs for a cycle point would cause cat-log to return the scheduler log rather than error.